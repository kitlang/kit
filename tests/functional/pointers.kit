function main() {
    var a = 1;
    var b: &Int = a;
    printf("%i\n", *b);

    var c = 2.5;
    var d: Ptr[Float] = c;
    printf("%.1f\n", *d);

    var e = b[0];
    printf("%i\n", e);
}
