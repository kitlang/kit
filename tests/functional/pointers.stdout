1
2.5
1
